### PR TITLE
replaced html anchor tag with its markdown counterpart in README.md

### PR DESCRIPTION
**Pull request by zahash**
replaced HTML anchor tag with its markdown counterpart; some other minor formatting improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ user@host:~$ sudo python3 packet_sniffer.py
 
 - Sample output captured during execution:
 
-```
+```sh
```
Comment (EONRaider) [inline README.md]: Can you remove this marker? It is highlighting parts of this code block as if they were shell scripts, which they are not.

File: README.md [hunk 2]
```diff
@@ -53,12 +53,12 @@ docker run --network host sniff python packet_sniffer.py --help
...
 
 Usage of `--network host` is not supported on OS X or Windows
-so this container won't be fully functional - but you will see packets 
+so this container won't be fully functional - but you will see packets
 traveling within the docker subnet.
 
 ## Usage
 
-```
+```sh
```
Comment (EONRaider) [inline README.md]: This marker too.

**Review by EONRaider — CHANGES_REQUESTED** [review]
Thanks for taking the time for this. Would you mind making those minor fixes I pointed in the code?

File: README.md (hunk 1 above)
Comment (zahash) [reply README.md]: Made a new commit with the required changes

File: README.md (hunk 2 above)
Comment (zahash) [reply README.md]: Made a new commit with the required changes

**Comment by zahash** [comment]
Made a new commit with the required changes

**Review by EONRaider — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (EONRaider) [reply README.md]: Done.

**Pull request merged**